app/views/ControlWindow.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @author Bingzheng Feng
+ * @module views/ControlWindow
+ * @requires models/Commands
+ * @requires util/Validator
+ * @requires util/Logger
+ * @requires models/TelemetryData
+ * @requires util/Template
+ * @requires electron
+ * @listens models/TelemetryData~TelemetryData:data_received
+ * @copyright Waterloo Aerial Robotics Group 2016
+ * @licence https://raw.githubusercontent.com/UWARG/WARG-Ground-Station/master/LICENSE
+ * @description Responsible for displaying control pitch, yaw, roll
```

Comment:
The description is misleading. Its actually reponsible for displaying all the received controller inputs